schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+ id               INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ ] [OPTIONAL] Kindly mark the id column as the primary key to uniquely identify each table record.
 ```
PRIMARY KEY (id)
```